src/endpoint/s3/s3_utils.js

```diff
@@ -322,6 +322,21 @@ function set_response_object_md(res, object_md) {
     }
 }
 
+/**
+ * @param {nb.S3Response} res 
+ */
+function set_response_supported_storage_classes(res) {
+    const storage_classes = [];
+    if (!config.DENY_UPLOAD_TO_STORAGE_CLASS_STANDARD) {
+        storage_classes.push('STANDARD');
+    }
+    if (config.NSFS_GLACIER_ENABLED && config.NSFS_GLACIER_LOGS_ENABLED) {
```

Comment:
no need to require logs enabled, it might be that we enable glacier with fs policies to handle the transitions rather than logging.
```suggestion
    if (config.NSFS_GLACIER_ENABLED) {
```

Reply:
Actually I added this intentionally because I found myself tripping over it. I enabled `NSFS_GLACIER_ENABLED` but my restores were not working. The reason was that there were no logs being written as the default is `false`. If we report `GLACIER` being supported then at least should truly be working, right?

Maybe we can make it backend specific (we get backend anyway from config) and then report it but I am inclined to keep this check here so as to not cause confusion.

Reply:
I still don't want to require enabling logging for that. NSFS_GLACIER_ENABLED is enough to tell if we support this class or not.